cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -7,6 +7,15 @@
 # Sanity Checks
 ################################
 
+# Ensure CUDA_TOOLKIT_ROOT_DIR is specified, since it is needed 
+# by the call to find_packag(CUDA) below.
```

Comment:
Typo: `find_packag` -> `find_package`

Reply:
Done - Resolved and amended previous commit.